doc/python/axes.md

```diff
@@ -388,7 +388,7 @@ You can add minor ticks to an axis with `minor`. This takes a `dict` of properti
 
 In the following example, we add minor ticks to the x-axis and then to the y-axis. For the y-axis we add ticks on the outside: `minor = {"ticks":"outside"}`. On the x-axis we've spcified some additional properties to style the minor ticks, setting the length of the ticks with `ticklen` and the color with `tickcolor`. We've also turned on grid lines for the x-axis minor ticks using `showgrid`.
 
-Note: Minor ticks and grid lines are not supported on multi-categorical, log, or 3D axes, or on color bars, ternary plots, polar charts, or geo.
+Note: Minor ticks and grid lines are not supported on color bars, ternary plots, polar charts, geo plots, or on multi-categorical, log, or 3D axes.
```

Comment:
The `log` is actually supported.

Reply:
Thanks. I'll add in an example

Reply:
https://github.com/plotly/plotly.js/blob/64a211ba79d07c37c06bc8183cd342f89bc5d1bd/test/jasmine/tests/axes_test.js#L6610-L6623

Reply:
`Note: Minor ticks and grid lines are not supported on ...` 


to


`Note: At the moment minor ticks and grid lines are not supported on ...`

Reply:
`Note: Minor ticks and grid lines are not supported on ...` 


to


`Note: At the moment minor ticks and grid lines are not supported on ...`